config/prow/cluster/ghproxy.yaml

```diff
@@ -64,6 +64,16 @@ spec:
         volumeMounts:
         - name: cache
           mountPath: /cache
+        livenessProbe:
```

Comment:
Please make these changes to the deployment file in a separate PR that we hold until the source changes to ghproxy have been merged, published, and deployed. Otherwise this will cause ghproxy to be considered unavailable in the meantime.

Reply:
Removed. I will open a follow-up to add them then.

Reply:
Thanks, this applies to the `readinessProbe` as well though I think.

Reply:
Thanks. Removed too.